### PR TITLE
updated index and intro to match spreadsheet, plus edits

### PR DESCRIPTION
**Pull request by LegoChemist**
Updated index and intro to match spreadsheet, plus new file on pump cannula changes, and final checks on BG smoothing.

### CODE REVIEW

File: docs/EN/Usage/Smoothing-Blood-Glucose-Data.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 
-Additionally, as of version 3.2, AAPS offers the option to smooth the data within AAPS. There are three options available in the [Config Builder](../Configuration/Config-Builder.md).
+Additionally, as of **AAPS** version 3.2, **AAPS** offers the option to smooth the data within **AAPS** rather than within the CGM app. There are three options available in the [Config Builder](../Configuration/Config-Builder.md).
 
 ![Smoothing](../images/ConfBuild_Smoothing.png)
 
 ## Exponential smoothing
 
-This is the recommended option to start with as it is most aggressive in resolving noise and rewrites the most recent value.
+This is the recommended option to start with, as it is most aggressive in resolving noise and rewrites the most recent value.
```
Comment (jbr7rr) [inline docs/EN/Usage/Smoothing-Blood-Glucose-Data.md:14]: Maybe expand the recommendation to : This is recommended when you have noisy data. Also note that exponential smoothing adds lag

File: docs/EN/5-DailyLifewithAAPS/DailyLife-PUMPS.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Disconnecting the pump for showering or activity
+
+If you take your pump off for showering, bathing, swimming, contact sports or other activities you must let **AAPS** know that no insulin is being delivered, to keep the IOB correct. The pump can be disconnected using the Loop Status icon on the **AAPS** Home Screen.
+
+As you are not getting any insulin while the pump is disconnected, you should reconnect every two hours to catch up for the missing basal. You can do this by connecting, bolusing the missing basal amounts (_e.g._ of the last two hours) before disconnecting again. This should help to avoid a severe lack of insulin which could result in diabetic ketoacidosis (DKA). If it is inconvenient to reconnect the pump during activity (cannula site is covered by wearing a wetsuit _etc._), consider a pen injection instead. This manual injection can be logged in **AAPS**, and doesn’t have to be logged at the time of injection (just make a note of the time of injection) since you can announce the insulin and backdate the time the insulin was actually given when you reconnect the pump.
```
Comment (jbr7rr) [inline docs/EN/5-DailyLifewithAAPS/DailyLife-PUMPS.md:47]: In dev, you can now log injection while pump is suspended:
https://github.com/nightscout/AndroidAPS/pull/3254

File: docs/EN/5-DailyLifewithAAPS/DailyLife-PUMPS.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Daily Life - Pumps
```
Comment (jbr7rr) [inline docs/EN/5-DailyLifewithAAPS/DailyLife-PUMPS.md:1]: This information seems most valid for tubed pumps? Is there anything to add for patch pumps?

Comment (LegoChemist) [reply docs/EN/5-DailyLifewithAAPS/DailyLife-PUMPS.md:1]: I don't have any expertise with patched pumps - we need someone else to add a section on this (are you able to?).

File: docs/EN/5-DailyLifewithAAPS/DailyLife-PUMPS.md (hunk 2 above)
Comment (LegoChemist) [reply docs/EN/5-DailyLifewithAAPS/DailyLife-PUMPS.md:47]: OK good to know - the main docs are just for master at present (there is a dev AAPS page in the docs though so you could make a note of it on there).

File: docs/EN/Usage/Smoothing-Blood-Glucose-Data.md (hunk 1 above)
Comment (LegoChemist) [reply docs/EN/Usage/Smoothing-Blood-Glucose-Data.md:14]: OK will do that in the next revision, thanks.

**Comment by peterleimbach** [comment]
@LegoChemist Thank you
 @jbr7rr Thank you

**Pull request merged**